autrainer/training/training.py

```diff
@@ -246,7 +246,7 @@ def __init__(
         _preprocess_pipe = SmartCompose([])
         _file_handler = self.data.file_handler
         _features_subdir = cfg.dataset.get("features_subdir", "default")
-        if _features_subdir != "default":
+        if not isinstance(self.data.file_handler, AudioFileHandler):
```

Comment:
I do not like the current check either, but it was necessary as now we allow other subdirs and not only default.

Reply:
Yeah this is a bit tricky, but since we are audio first it should be fine, but i also dont have a "long term" solution at the moment.

Reply:
It actually breaks things as it won't let `ToyTabular-C` etc. run through as it then looks for `preprocessing/default`...

Reply:
I settled for the even uglier:

```python
if (
            not isinstance(self.data.file_handler, AudioFileHandler)
            and _features_subdir != "default" 
        ):
```